templates/accordion.hbs

```diff
@@ -13,10 +13,12 @@
                 <div class="accordion-item-body-inner">
                     {{{a11y_text body}}}
                     {{#if _graphic}}
+                    {{#if _graphic.src}}
```

Comment:
nah, it's not possible, you have to nest them. the way helpers work doesn't facilitate operators.

null is a falsie, so #if _grahpic when _graphic === null will not execute.